### DISCUSSION TITLE
Blogpost #2

### DISCUSSION THREAD

**Discussion by stefanschfr** [lucariebel/online-shop#3] [General]
## SRS 
Diese Woche wurde das [SRS](https://github.com/lucariebel/online-shop/blob/main/doc/SoftwareRequirementsSpecification.md) initial erstellt.

**Comment by SchloesserJonas** [comment]
Hallo zusammen,
gute erste Gedanken!
Konntet ihr euch schon hinsichtlich eines Designkonzeptes (Farbkonzept, Layout, etc) einigen und bereits erste Fortschritte im Designprozess machen?

GaLiGrü 😊
Paula, Safae, Samuel, Valentin, Jonas

> Green Sprout 🌱

**Comment by Mimikry2** [comment]
Moin,

eure Software Requirements Specification macht einen sehr durchdachten Eindruck, besonders die detaillierte Beschreibung der Funktionalitäten und Use Cases gefällt uns. Es wäre spannend zu erfahren, wie ihr die Anforderungen priorisiert und ob ihr bereits konkrete Überlegungen zur Skalierbarkeit des Online-Shops angestellt habt. Wir freuen uns darauf zu sehen, wie sich euer Projekt weiterentwickelt!

Viele Grüße,
Tim, Dennis, Nico

**Comment by ChristophNie** [comment]
Hallo,
euer SRS gibt einen guten Überblick über die wichtigsten Anforderungen und Funktionen. Besonders gefällt mir die klare Struktur und die Wahl des Tech-Stacks.
Insgesamt ist das SRS sehr gut gelungen, nur die Sequenzdiagramme könnten noch etwas übersichtlicher sein. 
LG
Team Büroplatzverwaltung
